uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/mocks/data/option-group.data.js

```diff
@@ -863,6 +863,43 @@ define([
       'is_reserved': '0',
       'is_active': '1'
     }],
+    hrleaveandabsences_work_pattern_change_reason: [{
```

Comment:
With this change, is the mock data for `hrjc_revision_change_reason` still necessary?

Reply:
There is another implementation using `hrjc_revision_change_reason`. It will still be necessary for the test unless we want to use same collection for both.

Reply:
What other implementation? Shouldn't it be updated as well?

Reply:
According to the [ticket description](https://compucorp.atlassian.net/browse/PCHR-3451), `Work Pattern Change Reasons` currently uses a shared option group with `Contract Change Reason`. All this implementation does is to separate Work Pattern Change Reasons from the other. There is no need of updating since it is using the right change reasons.

Reply:
I don't understand @ajesamson. You said that there's another implementation using `hrjc_revision_change_reason`. What is this implementation?

Reply:
The [absence-tab-work-pattern-component](https://github.com/civicrm/civihr/blob/staging/uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/absence-tab/components/absence-tab-work-patterns.component.js) file has a [test file](https://github.com/civicrm/civihr/blob/staging/uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/absence-tab/components/absence-tab-work-patterns.component.spec.js) that is making reference to the `hrjc_revision_change_reason`.

Reply:
@ajesamson why this reference wasn't updated? Are we using this option group for something else?

Reply:
@davialexandre i actually have no idea if we are using it. I did not note the difference initially until i discovered my changes were not effected.